# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1491230676 14400
#      Mon Apr 03 10:44:36 2017 -0400
# Node ID b4f386ee712ffe9c6f93b65f012612184fa57c2e
# Parent  68dbde0e670aca894106672a7a9cb50e89085701
# Parent  4eff94123bea3ddcdba380ba88635e5fdc973ef5
maint: Periodic merge of stable to default.

diff --git a/doc/interpreter/install.txi b/doc/interpreter/install.txi
--- a/doc/interpreter/install.txi
+++ b/doc/interpreter/install.txi
@@ -282,16 +282,23 @@ interpreter.
 @item OpenGL
 API for portable 2-D and 3-D graphics (@url{http://www.opengl.org}).  An
 OpenGL implementation is required to provide Octave's OpenGL-based
 graphics functions.  Octave's OpenGL-based graphics functions usually
 outperform the gnuplot-based graphics functions because plot data can be
 rendered directly instead of sending data and commands to gnuplot for
 interpretation and rendering.
 
+@item PortAudio
+PortAudio (@url{http://www.portaudio.com/}) provides a very simple API
+for recording and/or playing sound using a simple callback function or
+a blocking read/write interface.  It is required for the audio processing
+functions @code{audioplayer}, @code{audiorecorder}, and
+@code{audiodevinfo}.
+
 @item Qhull
 Computational geometry library (@url{http://www.qhull.org}).  Qhull is
 required to provide the functions @code{convhull}, @code{convhulln},
 @code{delaunay}, @code{delaunayn}, @code{voronoi}, and @code{voronoin}.
 
 @item QRUPDATE
 QR factorization updating library
 (@url{http://sourceforge.net/projects/qrupdate}).  QRUPDATE is used to
@@ -310,16 +317,22 @@ It is a large framework, but the only co
 core, and network modules.
 
 @item SuiteSparse
 Sparse matrix factorization library
 (@url{http://www.cise.ufl.edu/research/sparse/SuiteSparse}).
 SuiteSparse is required to provide sparse matrix factorizations and
 solution of linear equations for sparse systems.
 
+@item SUNDIALS
+The SUite of Nonlinear and DIfferential/ALgebraic Equation Solvers
+(@url{http://computation.llnl.gov/projects/sundials}) is required for
+the Ordinary Differential Equations (ODE) solvers @code{ode15i} and
+@code{ode15s}.
+
 @item zlib
 Data compression library (@url{http://zlib.net}).  The zlib library is
 required for Octave's @code{load} and @code{save} commands to handle
 compressed data, including @sc{matlab} v5 MAT files.
 @end table
 
 @node Running Configure and Make
 @section Running Configure and Make
diff --git a/libgui/src/m-editor/file-editor-tab.cc b/libgui/src/m-editor/file-editor-tab.cc
--- a/libgui/src/m-editor/file-editor-tab.cc
+++ b/libgui/src/m-editor/file-editor-tab.cc
@@ -104,16 +104,19 @@ file_editor_tab::file_editor_tab (const 
 
   _edit_area = new octave_qscintilla (this);
   _line = 0;
   _col  = 0;
 
   _bp_lines.clear ();      // start with empty lists of breakpoints
   _bp_conditions.clear ();
 
+  // disable editor drag & drop so parent can handle
+  _edit_area->setAcceptDrops (false);
+
   connect (_edit_area, SIGNAL (cursorPositionChanged (int, int)),
            this, SLOT (handle_cursor_moved (int,int)));
 
   connect (_edit_area, SIGNAL (SCN_CHARADDED (int)),
            this, SLOT (handle_char_added (int)));
 
   connect (_edit_area, SIGNAL (SCN_DOUBLECLICK (int, int, int)),
            this, SLOT (handle_double_click (int, int, int)));
